jurigged/codetools.py

```diff
@@ -43,6 +47,27 @@ def get(self, item, dflt):
         return getattr(self.cls, item, dflt)
 
 
+def _get_future_compiler_flags(glb) -> int:
+    """use future feature names to get any that are set in the global namespace
+
+    `or` these together to get the flags for the compile function
+    """
+    flags = 0
+    for key in glb.keys() & _future_feature_names:
+        with suppress(Exception):
+            flags |= glb[key].compiler_flag
+    return flags
+
+
+def _compile_with_flags(node, mode, filename, glb):
+    """compile the node with the future flags set in the global namespace
+
+    basically, respect `from __future__ import annotations`
+    """
+    flags = _get_future_compiler_flags(glb)
+    return compile(node, mode=mode, filename=filename, flags=flags)
```

Comment:
I would suggest removing this from this PR since its already in a different one.